src/vaadin-grid-drag-and-drop-mixin.html

```diff
@@ -204,13 +204,20 @@
             // The grid is empty or "on-grid" drop mode was used, always default to "empty"
             this._dropLocation = DropLocation.EMPTY;
           } else if (!row || row.parentNode !== this.$.items) {
-            // The dragover didn't occur on a body row, default to "empty"...
-            this._dropLocation = DropLocation.EMPTY;
+            // The dragover didn't occur on a body row but the grid has items
+
+            if (row && this._effectiveSize) {
```

Comment:
`this._effectiveSize` seems unnecessary as it's always truthy at this point

Reply:
Done